telos-frontend/src/App.js

```diff
@@ -1,23 +1,12 @@
-import logo from './logo.svg';
 import './App.css';
+import Journal from './views/journal/Journal';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <div className="Journal-view">
+        <Journal/>
```

Comment:
Make sure you change the test for learn react link since you've removed it

Reply:
Thank you! Have run it now